pkg/jobstore/boltdb/store.go

```diff
@@ -937,28 +931,34 @@ func (b *BoltJobStore) updateJobState(tx *bolt.Tx, request jobstore.UpdateJobSta
 		}
 	}
 
-	return b.appendJobHistory(tx, job, previousState, request.Event)
+	return nil
+}
+
+// AddJobHistory appends a new history entry to the job history
+func (b *BoltJobStore) AddJobHistory(ctx context.Context, jobID string, events ...models.Event) error {
+	return b.update(ctx, func(tx *bolt.Tx) (err error) {
```

Comment:
Is it possible to batch these operations?
https://pkg.go.dev/github.com/boltdb/bolt#DB.Batch